test/src/tests-forwarders.js

```diff
@@ -2522,15 +2851,32 @@ describe('forwarders', function() {
             Object.keys(mParticle.getInstance()._preInit.integrationDelays)
                 .length
         ).equal(3);
+
+        window.mParticle.config.logLevel = 'verbose';
```

Comment:
Is this intentional or did you put this in for debugging?

Reply:
debugging.  i'll remove